version/version.go

```diff
@@ -1,6 +1,9 @@
 package version
 
 import (
+	_ "embed"
```

Comment:
Since we're passing in ldflags we can remove the go:embed changes from here and just update the default `Version` to ""